Controllers/OrderController.cs

```diff
@@ -0,0 +1,186 @@
+
+                }
+                return View(orders);
+            }
+            catch (Exception e)
+            {
+                ViewBag.ErrorMessage = "No Orders found.";
+                return View();
+            }
+
+        }
+        [HttpPost]
+        public IActionResult ConfirmOrders(Guid pizzaId)
+        {
+            try
+            {
+                var pizza = _pizzaAppDbContext.Pizza.FirstOrDefault(p => p.PizzaId == pizzaId);
+                if (pizza != null)
+                {
+                    var order = new Order
+                    {
+                        PizzaId = pizzaId
+                    };
+                    _pizzaAppDbContext.Order.Add(order);
+                    _pizzaAppDbContext.SaveChanges();
+                    return RedirectToAction("DisplayOrders");
+                }
+                else
+                {
+                    TempData["ErrorMessage"]= "This Pizza does not exist.";
+                    return RedirectToAction("DisplayOrders");
+                }
+                
+            }
+            catch (Exception e)
+            {
+                ViewBag.ErrorMessage = "Order was not confirmed.";
+                return View();
+            }
+
+        }
+        [HttpGet]
+        public IActionResult EditOrders(Guid pizzaId)
+        {
+            try
+            {
+                var pizza = _pizzaAppDbContext.Pizza.FirstOrDefault(p => p.PizzaId == pizzaId);
+                if (pizza != null)
+                {
+                    ViewBag.Bases = _pizzaAppDbContext.Base.ToList();
+                    ViewBag.Toppings = _pizzaAppDbContext.Toppings.ToList();
+                    ViewBag.Sizes = _pizzaAppDbContext.Size.ToList();
+                    var orderView = new EditOrderViewModel
+                    {
+                        PizzaId = pizzaId
+                    };
+                    return View(orderView);
+                }
+                else
+                {
+                    ViewBag.ErrorMessage = "Pizza was not found.";
+                    ViewBag.Bases = _pizzaAppDbContext.Base.ToList();
+                    ViewBag.Toppings = _pizzaAppDbContext.Toppings.ToList();
+                    ViewBag.Sizes = _pizzaAppDbContext.Size.ToList();
+                    var orderView = new EditOrderViewModel
+                    {
+                        PizzaId = pizzaId
+                    };
+                    return View(orderView);
+                }
+               
+
+            }
+            catch (Exception e)
+            {
+                TempData["ErrorMessage"] = "Pizza can not be editted due to some error.";
+                return RedirectToAction("DisplayOrders");
+            }
+
+        }
+        [HttpPost]
+        public IActionResult EditOrders(EditOrderViewModel edittedOrder)
+        {
+            try
+            {
+                if (edittedOrder.ToppingIds.Count == 0 || edittedOrder.ToppingIds.Count > 3)
```

Comment:
Instead of using '3' everywhere, we can use a constant so that if in future, we want to allow customers to select upto 4 or more toppings, we will not have to change this value everywhere in the code.